src/gas/gas.cpp

```diff
@@ -96,7 +101,8 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   const std::string eos_name = pin->GetOrAddString("gas", "eos", "ideal");
   if (eos_name == "ideal") {
     const Real gamma = pin->GetOrAddReal("gas", "gamma", 1.66666666667);
-    const Real cv = pin->GetOrAddReal("gas", "cv", 1. / (gamma - 1.));
+    const Real mmw = pin->GetOrAddReal("gas", "mmw", 1.);
+    const Real cv = constants.GetKBCode() / ((gamma - 1.) * mmw);
```

Comment:
This should be `mmw * amu`?

Reply:
Yeah better to specify this in units of `amu` than `g`. 

We might want to *also* allow the user to specify non-ideal cvs, specifically the Marshak EOS, so maybe we do want a `model` parameter here, e.g. `model = mmw, cv, marshak`. I'll try that.

Reply:
I'd suggest just doing, 

```
const Real mmw = pin->GetOrAddReal("gas", "mmw", 1.);
const Real cv = pin->GetOrAddReal("gas", "cv", constants.GetKBCode() / (mmw * constants.GetAMUCode() * (gamma - 1.)));
```

We could back out the `mmw` if the user specified just `cv`, but I think in cases like that `mmw` is probably meaningless. 

Also, can we call `mmw` either `mu` (which is what it's called in the astro literature) or `abar` (which is what it's called in the lab codes)?